deeposlandia/__init__.py

```diff
@@ -13,3 +15,12 @@
              "%(funcName)s : %(color)s%(message)s%(color_stop)s"))),
 ))
 logger = daiquiri.getLogger("root")
+
+_DEEPOSL_CONFIG = os.getenv('DEEPOSL_CONFIG')
+_DEEPOSL_CONFIG = _DEEPOSL_CONFIG if _DEEPOSL_CONFIG is not None else "config.ini"
+config = ConfigParser()
+if os.path.isfile(_DEEPOSL_CONFIG):
+    config.read(_DEEPOSL_CONFIG)
+else:
+    logger.error("No file config.ini!")
+    sys.exit(1)
```

Comment:
is the module `sys` imported somewhere?

Reply:
No, you're right, good catch! That's fixed!